content/riak/kv/2.2.3/developing/api/http/search-query.md

```diff
@@ -25,6 +25,8 @@ GET /search/query/<index_name>
 
 ## Optional Query Parameters
 
+Query parameters must be UTF-8 encoded.
```

Comment:
I did not mention percent-encoding as that is part of HTTP. Wanting to be extremely explicit and complte, and wanting to avoid any misunderstandings, this shall be e.g. "Query parameters must be UTF-8 encoded, before being percent-encoded (as required by HTTP)."